source/guide_screego.rst

```diff
@@ -0,0 +1,157 @@
+  [isabell@stardust ~]$ uberspace port add
+  Port 40131 will be open for TCP and UDP traffic in a few minutes.
+  [isabell@stardust ~]$
+
+Installation
+============
+
+Create a new directory, enter the directory you just created, download the latest version, unpack the archive, delete the archive afterwards, make the binary executable and create the file ``screego.config`` as a copy of the file ``screego.config.example``:
+
+.. note:: Replace ``1.11.1`` with the version of the `latest release`_.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mkdir ~/screego
+ [isabell@stardust ~]$ cd ~/screego
+ [isabell@stardust screego]$ wget "https://github.com/screego/server/releases/download/v1.11.1/screego_1.11.1_linux_amd64.tar.gz"
+ [isabell@stardust screego]$ tar xvf screego_1.11.1_linux_amd64.tar.gz
+ [isabell@stardust screego]$ rm screego_1.11.1_linux_amd64.tar.gz
+ [isabell@stardust screego]$ chmod +x screego
+ [isabell@stardust screego]$ cp screego.config.example screego.config
+ [isabell@stardust screego]$
+
+Configuration
+=============
+
+Configure Screego
+-----------------
+
+.. note:: You can use the following command to generate your own ``SCREEGO_SECRET``: ``tr -dc A-Za-z0-9 < /dev/urandom | head -c 40; echo``
+
+.. warning:: Replace ``isabell`` with your username, ``<YOUR_SECRET>`` with your secret and ``40130``/``40131`` with your ports!
+
+The following settings must be adjusted in the ``~/screego/screego.config``:
+
+::
+
+ SCREEGO_EXTERNAL_IP=dns:isabell.uber.space
+
+ SCREEGO_SECRET=<YOUR_SECRET>
+
+ SCREEGO_SERVER_TLS=true
+
+ SCREEGO_TLS_CERT_FILE=/readonly/isabell/certificates/isabell.uber.space.crt
+
+ SCREEGO_TLS_KEY_FILE=/readonly/isabell/certificates/isabell.uber.space.key
+ 
+ SCREEGO_SERVER_ADDRESS=0.0.0.0:40130
+ 
+ SCREEGO_TURN_ADDRESS=0.0.0.0:40131
+
+.. note::
+  Please have a look at "`NAT Traversal`_" on the official project page. In most cases STUN should be sufficient, but if TURN is used it is necessary to open more ports and specify them under ``SCREEGO_TURN_PORT_RANGE`` in the ``screego.config`` file. For the number of ports to be opened, the following currently applies approximately: ``shared video streams * users in the room * 4``
+  There is also a `bug in the TURN server`_ which results in not all connections being properly cleaned up. This means that even more ports may be required.
+
+Setup daemon
+------------
+
+Create ``~/etc/services.d/screego.ini`` with the following content:
+
+::
+
+ [program:screego]
+ directory=%(ENV_HOME)s/screego/
+ command=%(ENV_HOME)s/screego/screego serve
+ autostart=yes
+ autorestart=yes
+ startsecs=30
+             
+.. include:: includes/supervisord.rst
+
+If it's not in state RUNNING, check your configuration.
+
+Open website
+------------
+.. warning:: Replace ``isabell`` with your username and ``40130`` with your port!
+
+If Screego is running, you can find the website here:
+
+  * https://isabell.uber.space:40130/
```

Comment:
Is there a reason you're not configuring [a backend](https://manual.uberspace.de/web-backends/)? Their docs say:
> If TLS should be enabled for HTTP requests. Screego requires TLS,
> you either have to enable this setting or serve TLS via a reverse proxy.

So I would just disable TLS here, you then only have to open one port and serve the webinterface via backend. This solves another problem: On a fresh asteroid there is no certificate until the first request, screego fails to start in this case.

Reply:
You're right, I've changed it.